docs/user-guide/story-setting.md

```diff
@@ -1,13 +1,17 @@
+
  
- 
-The story settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in Edit Mode and can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
+The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in Edit Mode and can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
 
-<img src="../img/story-setting/story-setting.jpg" class="ms-docimage"/>
+<img src="../img/story-setting/story-setting-1.jpg" class="ms-docimage" style="max-width:400px"/>
 
 The editor can enable or disable and edit the following components:
 
 * The **Title** of the story, the default value is the title given to the story's resource in MapStore
 
+* The **Theme** to choose the background color and the text color of the whole StoryMap through the color picker by clicking the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/>, as follows: 
```

Comment:
* The **Theme** to choose the default background and text color of the whole story: clicking on the *Change Color* button <img src="../img/button/color-picker.jpg" class="ms-docbutton"/>  a color picker appears to allow selecting the desired color: